api/v1/types/model_types.go

```diff
@@ -31,3 +31,27 @@ type MetricResult struct {
 type MetricResultList struct {
 	Items []MetricResult `json:"items"`
 }
+
+type Stats struct {
+	Average    uint64 `json:"average"`
+	Percentile uint64 `json:"percentile"`
+	Max        uint64 `json:"max"`
+}
+
+type ExternalStatBundle struct {
+	Minute Stats `json:"minute"`
+	Hour   Stats `json:"hour"`
+	Day    Stats `json:"day"`
+}
+
+type StatsResponse struct {
+	// Uptime is in seconds
+	Uptime uint64                        `json:"uptime"`
```

Comment:
time.Time represents a time format, however the Uptime is a time.Duration field.
Since there is no standardized format for durations (unlike RFC3339 for time.Time), I opted to output the total number of seconds for that duration